### PR TITLE
Npm audit fix force

### PR DESCRIPTION
**Pull request by mstein-ck**

### CODE REVIEW

**Review by ymwymw — COMMENTED** [review]
Once you are working on this, can you fix the open dependabot issue as well?
Can be in a new PR

File: package.json [hunk 1]
```diff
@@ -1,16 +1,27 @@
 {
     "name": "cardknox-sdk-js",
     "version": "1.0.2201.1601",
-    "description": "",
+    "description": "A library for communicating with Cardknox supported terminals and Cardknox Gateway",
```
Comment (ymwymw) [inline package.json:4]:
```suggestion
    "description": "A library for communicating with Cardknox supported terminals and the Cardknox Gateway",
```

File: package.json [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
-    "description": "",
+    "description": "A library for communicating with Cardknox supported terminals and the Cardknox Gateway",
     "main": "dist/index.min.js",
+    "license": "",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/cardknox/cardknox-sdk-js.git"
+    },
+    "author": {
+        "email": "info@cardknox.com"
```
Comment (ymwymw) [inline package.json:12]: should we make this developersupport@cardknox as well?

**Comment by ymwymw** [comment]
> Once you are working on this, can you fix the open dependabot issue as well? Can be in a new PR

Let's see if the alert goes away after this PR.

File: package.json (hunk 2 above)
Comment (mstein-ck) [reply package.json:12]: Ok,
I just copied from React component

**Review by ymwymw — APPROVED** [review]

**Pull request merged**